src/Math-Tests-Complex/PMComplexTest.class.st

```diff
@@ -63,6 +50,18 @@ PMComplexTest >> testAddingTwoComplexNumbers [
 	self assert: c equals: 0 + 2 i
 ]
 
+{ #category : #'testing - arithmetic' }
+PMComplexTest >> testAdditionIsCommutativeWithAllRealNumbers [
```

Comment:
I've wondered whether we should add a test to demonstrate commutativity and associativity of addition. I think it is the definition of a (mathematical) field. Similarly, we might demonstrate the commutativity of multiplication, another part of the definition of a mathematical field.

@olekscode @SergeStinckwich @khinsen what are your thoughts. This could become a contract test for all fields, where addition and multiplication are commutative and associative.